adi/ad7405.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2025 Analog Devices, Inc.
+#
+# SPDX short identifier: ADIBSD
+
+from decimal import Decimal
+
+import numpy as np
+
+from adi.attribute import attribute
+from adi.context_manager import context_manager
+from adi.rx_tx import rx
+
+
+class ad7405(rx, context_manager):
+
+    """ AD7405 ADC """
+
+    _complex_data = False
+    channel = []  # type: ignore
+    _device_name = "ad7405"
+
+    def __init__(self, uri="", device_name="ad7405"):
```

Comment:
The default devices is ad7405, but emulation example is adum7701.  This needs to be changed to adum7701 to get the tests to work